### PR TITLE
Feature excel update term to unit

### PR DESCRIPTION
**Pull request by Etschbeijer**
Closses Issue #454

### CODE REVIEW

File: src/Client/SidebarComponents/Navbar.fs [hunk 1]
```diff
@@ -36,6 +36,17 @@ let private shortCutIconList model dispatch =
                 SpreadsheetInterface.CreateAnnotationTable ctrl |> InterfaceMsg |> dispatch
             )
         )
+        QuickAccessButton.create(
```
Comment (Freymaurer) [inline src/Client/SidebarComponents/Navbar.fs:39]: Converting between term and unit is a relatively rare usecase. I think it is better to keep the quicka ccess buttons for common use cases.

Comment (Etschbeijer) [reply src/Client/SidebarComponents/Navbar.fs]: fixed

**Comment by Etschbeijer** [comment]
The goal of this feature is the ability to convert building blocks.
You can convert terms into units and units into terms.
Additionally, you can also convert data into free text and free text into data.
You cannot convert input- and outpunt columns.

**Comment by Etschbeijer** [comment]
The buttons to use the convert feature are located in the building blocks area of the swate tab.
grafik

**Comment by Etschbeijer** [comment]
When you have selected a part of a building block in your table, you can klick on refresh. It will show you the type of building block you have chosen, e.g. Term. When it is convertible, you can click on "Convert XY to YX", converting the building block.
You do not have to click on refresh, you can directly click on "Convert XY to YX" and the button will be updated then.

**Comment by Etschbeijer** [comment]
You can convert different building blocks with the implementation of this feature but the kind of building block you can convert to depends on the building block type.
You can check what type your current building block has by clicking on the "Refresh" button in the building block in the excel-swate-nav-bar.
grafik

**Comment by Etschbeijer** [comment]
You can click he button below the refresh button to convert the currently selected building block into the building block type shown behind the button.
grafik

When you have selected another building block since you have selected the "Refresh" button, the buttons are not automatically updated. You can still convert your selected building block by pressen the "Convert" button, it will be updated then and show you what the building block used to be before you updated it.

**Pull request merged**